recipes/slow-buffer-to-buffer-alloc-unsafe-slow/README.md

```diff
@@ -0,0 +1,58 @@
+# SlowBuffer to Buffer.allocUnsafeSlow Codemod
+
+This codemod migrates deprecated `SlowBuffer` usage to `Buffer.allocUnsafeSlow()` to handle Node.js DEP0030.
```

Comment:
```suggestion
This codemod migrates deprecated `SlowBuffer` usage to `Buffer.allocUnsafeSlow()` to handle Node.js [DEP0030](https://nodejs.org/api/deprecations.html#DEP0030).
```

We link to the doc so when user read this readme on [Codemod Registry](https://app.codemod.com/registry) he can easily access to the deprecation.